legend-engine-language-pure-compiler/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/PureModel.java

```diff
@@ -439,6 +438,8 @@ private void loadOtherElementsPreStores(PureModelContextDataIndex pure)
 
     private void loadOtherElementsPostConnectionsAndRuntimes(PureModelContextDataIndex pure)
     {
+        // functions might reference runtime and connections
+        pure.functions.forEach(el -> visitWithErrorHandling(el, new PackageableElementSecondPassBuilder(this.getContext(el))));
```

Comment:
Can there be a negative side effect on moving this after connections and runtime?